dataflow_lite/include/dataflow_lite/cloudwatch/cloudwatch_service.h

```diff
@@ -229,6 +240,7 @@ void work() {
       if (Service::getState() == ServiceState::STARTED) {
         // publish mechanism via the TaskFactory
         task_to_publish->run(publisher_);
+        last_publish_time_ = getCurrentTimestamp();
```

Comment:
What if PutLogs wasn't even called? How do we know that something was actually published?

Reply:
we don't, but we do know that we are attempting to publish no more often than this time - if the publish fails for some reason, we still don't want to try publishing any sooner than the next tick of this timer (i think)